scripts/install.sh

```
@@ -0,0 +1,75 @@
+tar -xzf "$tmp/$archive" -C "$tmp" hookwise || err "failed to extract hookwise from $archive"
+
+# --- install ---------------------------------------------------------------
+mkdir -p "$BIN_DIR"
+install -m 0755 "$tmp/hookwise" "$BIN_DIR/hookwise" 2>/dev/null \
+  || { cp "$tmp/hookwise" "$BIN_DIR/hookwise" && chmod 0755 "$BIN_DIR/hookwise"; }
+
+printf 'Installed hookwise to %s/hookwise\n' "$BIN_DIR"
+
+case ":$PATH:" in
+  *":$BIN_DIR:"*) "$BIN_DIR/hookwise" --version ;;
+  # shellcheck disable=SC2016 # $PATH is intentionally literal in the printed hint
```

Shellcheck (0.11.0)

[error] Couldn't parse this case item. Fix to allow more checks.

(SC1073)